packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @fileoverview Enforces unbound methods are called with their expected scope.
+ * @author Josh Goldberg <https://github.com/joshuakgoldberg>
```

Comment:
Sorry we've changed to using all-contributors now and removed all other occurrences of these file-level author comments, please could you remove this